### PR TITLE
mbuffer: update to 20190127

### PR DESCRIPTION
**Pull request by ndowens**

### CODE REVIEW

File: srcpkgs/mbuffer/template [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Template file for 'mbuffer'
 pkgname=mbuffer
-version=20190119
+version=20190127
 revision=1
 build_style=gnu-configure
+conf_files="/etc/mbuffer.rc"
```
Comment (maxice8) [inline srcpkgs/mbuffer/template]: please keep them where is

**Comment by maxice8** [comment]
On Tue, Jan 29, 2019 at 04:28:33AM -0800, Nathan wrote:
> ndowens commented on this pull request.
> 
> 
> 
> >  revision=1
>  build_style=gnu-configure
> +conf_files="/etc/mbuffer.rc"
> 
> Ok. Moved it because xlint said it was wrong 
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/7951#discussion_r251811605

The last git should be ok as far as i remember.

Moving those variables that are not commonly used up top just because `xnew` does it was a dumb mistake, also moving them around cause git blame to lose track of who did what.

**Comment by ndowens** [comment]
Will revert it how it was just in a sec

**Comment by ndowens** [comment]
Fixed

**Comment by biopsin** [comment]
short_desc="Buffer data steams with many additional functions"
..streams is misspelled

**Pull request merged**